app/src/main/res/values-de/strings.xml

```diff
@@ -204,50 +203,50 @@
-    <string name="theme_custom">Custom Theme</string>
+    <string name="theme_default">Standard</string>
+    <string name="theme_paper">Papier</string>
+    <string name="theme_graph_paper">Karopapier</string>
+    <string name="theme_light">Hell</string>
+    <string name="theme_paper_light">Papier Hell</string>
+    <string name="theme_graph_paper_light">Karopapier Hell</string>
+    <string name="theme_high_contrast">Hoher Kontrast</string>
+    <string name="theme_inverted_high_contrast">Invertierter Hoher Kontrast</string>
+    <string name="theme_custom">Eigenes Thema</string>
 
-    <string name="screen_custom_theme">Custom Theme colors</string>
-    <string name="screen_custom_theme_summary">Customize colors of the Custom Theme.</string>
-    <string name="screen_custom_theme_summary_disabled">Choose \"Custom Theme\" in \"Theme\" setting above to enable this section.</string>
+    <string name="screen_custom_theme">Eigene Themenfarben</string>
+    <string name="screen_custom_theme_summary">Verändere Farben des eigenen Themas.</string>
+    <string name="screen_custom_theme_summary_disabled">Wählen Sie \"Eigenes Thema\" in der Themenauswahl, um diese Einstellungen zu Aktivieren.</string>
 
-    <string name="default_lineColor">Line</string>
-    <string name="default_lineColor_summary">Border color around a cell.</string>
+    <string name="default_lineColor">Rahmen</string>
+    <string name="default_lineColor_summary">Rahmen um eine einzelne Zelle</string>
 
-    <string name="default_sectorLineColor">Sector line</string>
-    <string name="default_sectorLineColor_summary">Border color around a block of 3x3 cells.</string>
+    <string name="default_sectorLineColor">Blockrahmen</string>
+    <string name="default_sectorLineColor_summary">Rahmen um einen 3x3 Zellblock</string>
 
     <string name="default_textColor">Text</string>
-    <string name="default_textColor_summary">Text color of an editable cell.</string>
+    <string name="default_textColor_summary">Textfarbe einer editierbaren Zelle</string>
 
-    <string name="default_textColorReadOnly">Read-only text</string>
-    <string name="default_textColorReadOnly_summary">Text color of a non editable cell.</string>
+    <string name="default_textColorReadOnly">Feste Zahl</string>
+    <string name="default_textColorReadOnly_summary">Textfarbe einer nicht-editierbaren Zelle</string>
 
-    <string name="default_textColorNote">Note text</string>
-    <string name="default_textColorNote_summary">Color of a note text of an editable cell.</string>
+    <string name="default_textColorNote">Notiztext</string>
+    <string name="default_textColorNote_summary">Textfarbe der Notizen</string>
 
-    <string name="default_backgroundColor">Primary background</string>
-    <string name="default_backgroundColor_summary">Background color of odd groups of 3x3 cells.</string>
+    <string name="default_backgroundColor">Erster Hintergrund</string>
```

Comment:
really "Erster" ???
why not use "Primärer" or "Vorrangiger"

Reply:
I even think primary/secondary like in the english translation is misleading. Naming it odd/even would fit at best IMO. The "primary background" has nothing what designates it to be primary.